<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_addBookButton">
       <property name="text">
        <string>添加书籍</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_logoutButton">
       <property name="styleSheet">
        <string notr="true">background-color: red</string>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="BookInfoViewForAdmin" name="m_bookInfoView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookInfoViewForAdmin</class>
   <extends>QTableView</extends>
   <header>view/bookinfoviewforadmin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
